libraries.json

```
@@ -323,5 +323,18 @@
     "source": "jakubpawlina/graphs.excalidrawlib",
     "preview": "jakubpawlina/graphs.png",
     "date": "2021-02-02"
-  }
+  },
+  {
+      "name": "Lo-Fi Wireframing Kit",
+      "description": "A collection of lo-fi wireframing components.",
+      "authors": [
+        {
+          "name": "Aleksandra Lazovic",
+          "url": "https://spfr.co"
+        }
+      ],
+      "source": "spfr/Lo-Fi_Wireframing_Kit.excalidrawlib",
+      "preview": "spfr/Lo-Fi_Wireframing_Kit.png",
```

can we rename them to use lower case?
```suggestion
      "source": "spfr/lo-fi-wireframing-kit.excalidrawlib",
      "preview": "spfr/lo-fi-wireframing-kit.png",
```

Also you'll need to rename the actual files.